src/main/java/org/jpeek/Report.java

```diff
@@ -130,15 +139,36 @@
      * @param mean Mean
      * @param sigma Sigma
      * @checkstyle ParameterNumberCheck (10 lines)
+     * @todo #135:30min Make Report save 'metric.xsd' itself. Currently, Report
+     *  is setting the schema's location on the output metric XML based on the
+     *  assumption that App will always save it to a relative subdirectory
+     *  'xsd' and with the same filename 'metric.xsd'. This promise is best
+     *  kept if Report class has the responsibility of saving metric.xsd itself.
      */
+    @SuppressWarnings("unchecked")
     Report(final XML xml, final String name,
         final Map<String, Object> args,
         final double mean, final double sigma) {
         this.skeleton = xml;
         this.metric = name;
-        this.params = args;
+        this.params = new MapOf<>(
+            // @checkstyle LineLength (1 line)
```

Comment:
@llorllale Can you indent this code so you don't have to escape checkstyle?

Reply:
@amihaiemil fixed